src/client/LumClient.ts

```diff
@@ -274,7 +273,7 @@ export class LumClient {
      * @param doc document to sign
      */
     signTx = async (wallet: LumWallet | LumWallet[], doc: LumTypes.Doc): Promise<Uint8Array> => {
-        let wallets: LumWallet[] = [];
+        let wallets: LumWallet[];
```

Comment:
Any reason why this was deleted?

Reply:
It's a redundant initializer, indeed, this variable is initialized in all cases just below

Reply:
Revamp all function